### PR TITLE
X11: fix pointer/integer type mismatch

### PR DESCRIPTION
**Pull request by floppym**
Fixed on master in 2da280b8a1748052b70b3f5a1ef0d8e932c33adc.

### CODE REVIEW

File: client/X11/xf_graphics.c [hunk 1]
```diff
@@ -438,7 +438,7 @@ static BOOL xf_Pointer_New(rdpContext* context, rdpPointer* pointer)
 
 #endif
 fail:
-	WLog_DBG(TAG, "%s: %ld", __func__, rc ? pointer : -1);
+	WLog_DBG(TAG, "%s: %ld", __func__, rc ? pointer : NULL);
```
Comment (akallabeth) [inline client/X11/xf_graphics.c:441]: you might take another look at the change on master, the format string is wrong.

**Review by akallabeth — CHANGES_REQUESTED** [review]

File: client/X11/xf_graphics.c (hunk 1 above)
Comment (floppym) [reply client/X11/xf_graphics.c:441]: Ah, I was looking at the wrong change. 2da280b8a1748052b70b3f5a1ef0d8e932c33adc is the relevant commit.

Comment (akallabeth) [reply client/X11/xf_graphics.c:441]: you copied it wrong ;)

**Review by akallabeth — APPROVED** [review]
LGTM

**Comment by akallabeth** [comment]
@freerdp-bot test please
@floppym thank you!

**Comment by akallabeth** [comment]
@freerdp-bot test please

**Pull request merged**